packages/fx-core/templates/core/v3Migration/csharp.app.yml

```diff
@@ -0,0 +1,93 @@
+version: 1.0.0
```

Comment:
@IvanJobs @Siglud please help review the content in this app yml template, the syntax is handlebars syntax (similar to mustache)